WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt

```diff
@@ -3,6 +3,7 @@ package org.wordpress.android.viewmodel.posts
 import org.apache.commons.text.StringEscapeUtils
 import org.wordpress.android.BuildConfig
 import org.wordpress.android.R
+import org.wordpress.android.R.string
```

Comment:
I am not sure if we should be making this change 🤔 I know it is used in other places within the codebase but it seems unrelated to the task at hand here. If it's a case where the CI is failing due to it then I guess it's fine to proceed as it might be a new rule I am not aware of. Let me know your thoughts.

Reply:
Android Studio function extraction did this and I didn't notice. When I read your comment I was surprised by it as well. I'm fixing this!